tokens/mainnet.json

```diff
@@ -1,5 +1,14 @@
 {
   "networkId": 0,
-  "tokens": [],
+  "tokens": [
+    {
+      "id": "c0c0af7a481e3e50c50e418bf8ff6923dc4d878ac3744474e8c708a8adccfb00",
+      "name": "Alf",
+      "symbol": "ALF",
+      "decimals": 0,
+      "description": "Alf, first token deployed on the mainnet",
+      "logoURI": "https://raw.githubusercontent.com/sven-hash/alf/master/alf.png"
```

Comment:
it has to be: 

```https://raw.githubusercontent.com/alephium/tokens-meta/master/logos/ALF.png```

to pass [our test](https://github.com/alephium/token-list/blob/master/test/token-list/token-list.test.ts#L46)

Also you need to submit the image in the PR